### ISSUE TITLE
Long-Term Vision: Merge into Plex Media Server for easy deployment and auto-configuration

### ISSUE THREAD

**Issue by epheterson** [wnielson/Plex-Remote-Transcoder#54]
I love the concept of this project, though it's very clearly a sophisticated, complicated, fragile system that puts off most users who may benefit from remote transcoding. 

Since Plex Media Server is easily available on many systems (i.e. simple UI on Mac/Windows and one-click installable on many NAS systems), it may make sense to merge this code into Plex Media Server, then develop simple UI to allow users to link PMS installations together for transcoding.

Users would have to point to the same media directories on both sides, then pretty much step back and watch it work.

**Comment by epheterson** [comment]
This also makes it easy to ensure PMS and PRT are updated together, and introduces a messaging UI that PRT can take advantage of, which is already familiar to users (Status -> Alerts view).

**Comment by wnielson** [comment]
Agreed, this would be fantastic to build into Plex itself.  Ideally, the functionality would be integrated into `Plex Media Server`, but unfortunately that is closed source and I am not a Plex employee.  In theory, it would be possible to add much of this functionality into the `Plex Transcoder`, which is open source.  The main issue is that the code is in c which, while not a technical barrier, takes longer development time and adds complexity to releases as the code would have to be compiled for the many architectures that Plex targets.

We'll see what happens in the future.
